fas2ipa/cli.py

```diff
@@ -171,6 +171,14 @@ def migrate_group(group, ipa):
     )
     try:
         ipa.group_add(name, **group_args)
+        for agreement_details in config.get("agreements").values():
```

Comment:
In toml, you represent a list of dicts as such:
```
[[agreements]]
name = "agreement 1"
key = "value"

[[agreements]]
name = "agreement 2"
key = "value"
```
(note the double brackets)
This seems to be a better data structure for what you want to do.